test/util/range.cpp

```diff
@@ -213,6 +213,21 @@ void test_pointers()
     BOOST_CHECK(bgr::at(r2, 5) == 5);
 }
 
+void test_const_pointers()
```

Comment:
Big picture:
* I have to admit I am not a Jamfile savvy, so I will leave it to the others to comment on it.
* Are you talking about `BOOST_AUTO_TEST_CASE` etc.? I already use this in the unit tests I have written from scratch.
* Please **DO NOT** use the prebuild Boost.Test framework. Boost.Geometry is a header only library, including its tests (and this is hard requirement). Relying on the prebuild Boost.Test framework would violate the header-only requirement.

Benefits:
* I agree.
* I thought this was possible already. I am probably missing something.
* True. We prefer to sacrifice compilation speed for header-only.